web/pages/index.tsx

```diff
@@ -3,19 +3,28 @@ import Head from "next/head";
 import { auth } from "../utils/firebase/firebaseClient";
 import { trpc } from "../utils/trpc";
 import { useAuth } from "../context/auth";
+import { mongo } from "mongoose";
 
 export default function Home() {
   async function handleLogin() {
     const provider = new FacebookAuthProvider();
     const result = await signInWithPopup(auth, provider);
     const user = result.user;
+
+    fetch("/api/users", {
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json",
+      },
+      body: JSON.stringify(user),
+    });
```

Comment:
After you make your tRPC endpoint, use the tRPC-provided hooks to get the data instead of writing your own fetches.